eth/mon2/types.go

```diff
@@ -143,7 +143,7 @@ func (m *Monitor) updateBlkNum() {
 	// don't lock upfront in case rpc takes long time
 	blkNum, err := m.ec.BlockNumber(context.Background())
 	if err != nil {
-		log.Warnln("get blknum err:", err)
+		log.Warnf("chain %d get blknum err: %w", m.chainId, err)
```

Comment:
minor: use %s when print error, %w is for fmt.Errorf err wrap

Reply:
for error fmt should be %v, not %s (although the difference is very subtle, mainly if err is nil)

https://pkg.go.dev/fmt